apps/console/src/app/(protected)/entities/page.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { PageHeading } from '@repo/ui/page-heading'
 
 const Page: React.FC = () => {
-  return <PageHeading heading="Entities" />
+  return <PageHeading heading="Vendors" />
```

Comment:
`Entities` is correct one here
```suggestion
  return <PageHeading heading="Entities" />
```